backend/src/core/enums/genre.py

```diff
@@ -0,0 +1,11 @@
+from src.core.enums.base import BaseEnum
+
+
+class GenreTypeEnum(BaseEnum):
+    COMEDY = "comedy"
+    TRAGEDY = "tragedy"
+    DRAMA = "drama"
+    FARCE = "farce"
+    MUSICAL = "musical"
+    THRILLER = "thriller"
+    POLITICAL = "political"
```

Comment:
Do we need to make first letter capital one? It's a question for @dmitryzhurkovsky mostly